index.js

```diff
@@ -191,8 +191,10 @@ function init (options) {
   var aliases = npmPackage._moduleAliases || {}
 
   for (var alias in aliases) {
-    if (aliases[alias][0] !== '/') {
-      aliases[alias] = nodePath.join(base, aliases[alias])
+    if (aliases[alias][0] !== '/') {		
```

Comment:
If aliases[alias] is an array, then you should do a different check on aliases[alias][0].